demo/demoproject/tests.py

```diff
@@ -10,6 +11,16 @@
 
 from demoproject._compat import decode
 
+from demoproject.models import Meter
+import os
+import sys
+
+# breakpoint()
+_parent = os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+sys.path.append(os.path.join(_parent, 'chartjs'))
+import util
```

Comment:
This part is kind of ugly, but this was about as clean as I could get it.

Reply:
See my Makefile change suggestions